user-guide/Reference/Software_support_life_cycles.md

```diff
@@ -47,7 +47,7 @@ Important announcements about the latest changes to DataMiner core functionality
 | Jobs app | End of Engineering | Module is being retired. | End of Life as of DataMiner version 10.5.x (Q4 2024) |
 | Child bookings | End of Engineering | Child bookings are being retired, as the same functionality is supported with contributing bookings | End of Life as of DataMiner version 10.5.x (Q4 2024) |
 | Recurring bookings | End of Engineering | Recurring bookings are being retired. Instead, we suggest duplicating the previous occurrence of the booking. | End of Life as of DataMiner version 10.5.x (Q4 2024) |
-| Legacy Reporter & Dashboards | End of Life | Module has been retired and replaced with the new DataMiner Dashboards app. | End of Life as of DataMiner version 10.4.x (Q4 2023) |
+| Legacy Reporter & Dashboards | End of Support | Module is being retired and replaced with the new DataMiner Dashboards app. | End of Life as of DataMiner version 10.6.x (Q4 2025) |
```

Comment:
@srskyline, I changed the current status back to "End of Support" and also updated the future status to "End of Life as of DataMiner version 10.6.x (Q4 2025)". Is this OK to merge?